src/main/kotlin/exceptions/ejercicio1.kt

```diff
@@ -0,0 +1,16 @@
+package org.example.exceptions
+
+fun main (){
+    try {
+        val numerador = 10
+        val divisor = 0
+        val result = numerador / divisor
+
+    }
+    catch (e: Exception){
+        println("La división por 0 es inválida, Error: $e")
```

Comment:
el mensaje debe ser este

```suggestion
        println("Excepción capturada: División por cero no permitida.")
```